homeassistant/components/husqvarna_automower_ble/config_flow.py

```diff
@@ -108,23 +123,15 @@ async def async_step_user(
             if not _pin_valid(user_input[CONF_PIN]):
                 errors["base"] = "invalid_pin"
             else:
-                self.address = user_input[CONF_ADDRESS]
+                self.address = format_mac(user_input[CONF_ADDRESS])
```

Comment:
I still think we need a config entry migration of existing entries, otherwise the unique id collision check may fail.

Bump config entry version to 1.2, then implement a migrator similar to this: https://github.com/home-assistant/core/blob/1b9acdc2334e1ffc564d8702ac9a8b749318788b/homeassistant/components/derivative/__init__.py#L63-L77

Reply:
Would this mean we need to migrate IDs?

Reply:
OK, let's remote the normalization of the MAC-address from this PR and move that to a separate PR 👍